src/pocketmine/entity/Skin.php

```diff
@@ -120,6 +152,48 @@ public function getGeometryData() : string{
 		return $this->geometryData;
 	}
 
+	/**
+	 * @return string
+	 */
+	public function getAnimationData() : string{
+		return $this->animationData;
+	}
+
+	/**
+	 * @return bool
+	 */
+	public function getPremium() : bool{
+		return $this->premium;
+	}
+
+	/**
+	 * @return bool
+	 */
+	public function getPersona() : bool{
+		return $this->persona;
+	}
+
+	/**
+	 * @return bool
+	 */
+	public function getCapeOnClassic() : bool{
+		return $this->capeOnClassic;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function getCapeId() : string{
+		return $this->capeId;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function getFullSkinId() : string{
+		return $this->skinId . "_" . $this->capeId;
```

Comment:
You could leave an empty string here, it is useless anyway.
However, I want to note that the client sends SkinId here again.

Reply:
The cape ID sent is (currently) always empty.